lib/std/multi_array_list.zig

```diff
@@ -93,6 +93,17 @@ pub fn MultiArrayList(comptime T: type) type {
                 }
             }
 
+            pub fn setRange(self: *Slice, index: usize, count: usize, elem: T) void {
+                const e = switch (@typeInfo(T)) {
+                    .Struct => elem,
+                    .Union => Elem.fromT(elem),
+                    else => unreachable,
+                };
+                inline for (fields, 0..) |field_info, i| {
+                    @memset(self.items(@as(Field, @enumFromInt(i)))[index..(index + count)], @field(e, field_info.name));
```

Comment:
```suggestion
                    @memset(self.items(@as(Field, @enumFromInt(i)))[index..][0..count], @field(e, field_info.name));
```

[slice by length](https://ziglang.org/download/0.11.0/release-notes.html#Slicing-By-Length)